kivymd/uix/spinner.py

```diff
@@ -74,7 +75,7 @@ class MDSpinner(ThemableBehavior, Widget):
     defaults to True
     """
 
-    color = ListProperty([])
+    color = ListProperty(ThemeManager().primary_color)
```

Comment:
ThemeManager object will be created on each initialization of MDSpinner. It is not good

Reply:
Oh, sorry. You're right. I think there's another approach:
We can change the inheritance of the MDSpinner to only ThemeManager and replace all the self.theme_cls to only self. This way, the line 78 can be kept as the original one (```color = ListProperty([])```) and the code works fine.